chart/values.schema.json

```diff
@@ -2153,8 +2153,64 @@
                 }
             }
         },
+        "_ports": {
+            "description": "All container ports used by chart.",
+            "type": "object",
+            "x-docsSection": "Ports",
+            "additionalProperties": false,
+            "properties": {
+                "flowerUI": {
+                    "description": "Flower UI port.",
+                    "type": "integer",
+                    "default": 5555,
+                    "const": 5555
+                },
+                "airflowUI": {
+                    "description": "Airflow UI port.",
+                    "type": "integer",
+                    "default": 8080,
+                    "const": 8080
```

Comment:
I know this will block changes to the port when values.yaml is validated with values.schema.yaml, but having these "hard coded" in values.yaml where people are expecting to find changeable parameters would definitely be confusing.

If we do want to force them to use a certain container port, we should just hard code it in the templates instead (and in most cases we are talking about 1, maybe 2 places if we use named ports in services). It'd be easy to block attempted changes via `config.webserver.web_server_port`, but env vars are harder to comprehensively block. Another plus, our values is big enough as it is.